src/components/DialoguesWithoutCuratorButton.tsx

```diff
@@ -0,0 +1,45 @@
+import { ChatBubble } from '@mui/icons-material';
+import { Box, Button } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { TicketSearchParam } from '../api/searchApi';
+import { TicketStatus } from '../types/publication_types/ticket.types';
+import { dataTestId } from '../utils/dataTestIds';
+
+export const DialoguesWithoutCuratorButton = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const statusNew: TicketStatus = 'New';
```

Comment:
flisespikking, man kanskje flytte denne utenfor/over komponenten, siden den er lik for hver render uansett?

Reply:
Jeg er all for flisespikking! Fikser